features/mesh/dataplanes/DataplanePolicies.feature

```diff
@@ -305,10 +313,16 @@ Feature: Dataplane policies
 
       When I visit the "/meshes/default/data-planes/dataplane-1/policies" URL
 
-      Then the "$standard-dataplane-policies" element exists
+      Then the "$legacy-policies" element doesn't exist
+      And the "$sidecar-dataplane-policies" element doesn't exist
+      And the "$builtin-gateway-dataplane-policies" element doesn't exist
```

Comment:
Ok sounds good, thanks for adding that, as long as we look for something I think thats the most important thing.

---

I considered changing the "doesn't exist" step to force you to check for something first/also. Sounds like it would still be a good idea to do that.

Reply:
> I considered changing the "doesn't exist" step to force you to check for something first/also. Sounds like it would still be a good idea to do that.

They all do that already.

Reply:
Right yeah, but only becauase I asked during review tho right. I'm saying something like only having:

`And the "$expected" element exists but the "$unexpected" doesn't exist`

And removing the existing step:

`And the "$unexpected" element doesn't exist`

Then there would be no way to check for non-existence without making sure something is on the page first.